app/components/instructorgroup-details.js

```diff
@@ -1,24 +1,22 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { dropTask } from 'ember-concurrency';
 
-export default Component.extend({
-  tagName: '',
+export default class InstructorgroupDetailsComponent extends Component {
+  @dropTask
+  *addUser(user) {
+    const users = yield this.args.instructorGroup.users;
```

Comment:
I'm fairly certain this was a workaround for old Ember Data and only one side of this relationship needs to be changed and saved.

Reply:
handled. :heavy_check_mark: